src/server/database/mongodb-adaptor.js

```diff
@@ -8,7 +8,9 @@ class MongoCollectionAdaptor {
   }
 
   put (doc) {
-    return this.collection.insertOne(doc)
+    return new Promise((resolve, reject) => {
+      this.collection.insertOne(doc).then(() => resolve(doc)).catch((err) => reject(err))
+    })
```

Comment:
If inserOne returns a promise, why is there the need to rewrap it inside a promise?